08-rails/active-record-models.md

```diff
@@ -192,69 +188,68 @@ SQL (0.3ms)  UPDATE "students" SET "pie" = ?, "motto" = ?, "updated_at" = ? WHER
 |:---------------------------------|:-----------------------------------|:-----------|:-----------------------|:------------|
-| `Student.all`                    | Get all table entries              | N/A        | Collection of Students | Yes         |
-| `Student.first`, `Student.last`  | Get entry with lowest / highest ID | N/A        | One Student            | Yes         |
-| `Student.find(4)`                | Get the entry in row 4             | Integer    | One Student            | Yes         |
-| `Student.where(name: 'libby')`   | Get entries that match             | Hash       | Collection of Students | Yes         |
-| `Student.find_by(name: 'libby')` | Get matching entry with lowest ID  | Hash       | One Student            | Yes         |
-| `Student.order(:name)`           | Sort entries                       | Field Name | Collection of Students | Yes         |
-| `Student.count`                  | Count entries                      | N/A        | Integer                | Yes         |
-| `Student.size`                   | Count entries                      | N/A        | Integer                | Memoized    |
-| `Student.empty?`                 | Are there zero entries?            | N/A        | Boolean                | Memoized    |
+| `Book.all`                    | Get all table entries              | N/A        | Collection of Books | Yes         |
+| `Book.first`, `Book.last`  | Get entry with lowest / highest ID | N/A        | One Book            | Yes         |
+| `Book.find(4)`                | Get the entry in row 4             | Integer    | One Book            | Yes         |
+| `Book.where(author: 'so and so')`   | Get entries that match             | Hash       | Collection of Books | Yes         |
+| `Book.find_by(title: 'such and such')` | Get matching entry with lowest ID  | Hash       | One Book            | Yes         |
+| `Book.order(:title)`           | Sort entries                       | Field Name | Collection of Books | Yes         |
+| `Book.count`                  | Count entries                      | N/A        | Integer                | Yes         |
+| `Book.size`                   | Count entries                      | N/A        | Integer                | Memoized    |
+| `Book.empty?`                 | Are there zero entries?            | N/A        | Boolean                | Memoized    |
 
 ### Interacting with a Retrieved Entry
-Assume that `libby` is a local copy of an entry in the table, as returned by a call like `Student.first`. Remember that any changes made to a local copy will not be reflected in the database until `save` is called on that copy.
+Assume that `libby` is a local copy of an entry in the table, as returned by a call like `Book.first`. Remember that any changes made to a local copy will not be reflected in the database until `save` is called on that copy.
```

Comment:
We should use `my_book` instead of `libby` here to match the examples in our table below.